### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by shivshankar9**

### CODE REVIEW

**Comment by shivshankar9** [comment]
Contribution guidelines

**Comment by sameersrivastava13** [comment]
> Contribution guidelines

Looks good to me @shivshankar9. Just rename the file to contributors.md and submit the PR.

**Comment by shivshankar9** [comment]
Ok give me sec

File: README.md [hunk 1]
```diff
@@ -50,6 +50,90 @@ python app.py
...
+
+**14.** Click on `Create Pull Request`.
+
+**15.** Voila :exclamation: You have made a PR to the  sameersrivastava13/Vaccine-Tracker-App project :boom: . Sit back patiently and relax while the project maintainers review your PR. Please understand, at times the time taken to review a PR can vary from a few hours to a few days.
+
+<p align="center"><img src="https://media.tenor.com/images/b562ddcfb131e962f9dfa01bd32a30d1/tenor.gif" width=30%></p>
+
+## Project Admin 👨: sameersrivastava13
+
+
```
Comment (sameersrivastava13) [inline README.md]: @shivshankar9 you have repeated the same` contribution.md` file information in `readme.md` file. Please remove the entire part from the readme.md file. Rest everything looks fine to me.

**Review by sameersrivastava13 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (shivshankar9) [reply README.md]: Ok

**Review by shivshankar9 — COMMENTED** [review]
Removed

**Comment by sameersrivastava13** [comment]
@shivshankar9 Ok great. Everything looks fine to me now. 
@noviicee Would you like to add something or review once? or should I merge the PR.

File: README.md [hunk 2]
```diff
@@ -50,6 +50,7 @@ python app.py
 
 5. Go to ` http://127.0.0.1:5000/` and enjoy the application.
 
+
```
Comment (noviicee) [inline README.md]: Is there any particular need of this extra line here..?

Comment (noviicee) [reply README.md]: Otherwise LGTM :rocket:

Comment (sameersrivastava13) [reply README.md]: Actually I also noticed the same. I'll remove that extra line and will merge then.

**Pull request merged**